### ISSUE TITLE
hyperparameter tuning with keras on GCP: failing for no reason

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_multilingual_text_classification#40]
everything is setup properly
the special metric is Keras in created
the metric is display in TensorBoard
hp tuning doesn't seems to get this metric

GCP support is looking at this issue

**Comment by tarrade** [comment]
temporary fix by creating an instance of HP engine and passing the metric directly.

The issue seems to be that even is there is an env variable with this path /var/hypertune/output.metric such file is not created and filled with the info from the callback from Keras.  Using http://hpt.report_hyperparameter_tuning_metric will create this file.

**Comment by tarrade** [comment]
GCP is working on a solution but we have a solution  working so closing since it has no impact.
